Sources/Public/Animation/AnimationViewInitializers.swift

```diff
@@ -26,7 +26,7 @@ extension AnimationView {
     animationCache: AnimationCacheProvider? = LRUAnimationCache.sharedCache)
   {
     let animation = Animation.named(name, bundle: bundle, subdirectory: nil, animationCache: animationCache)
-    let provider = imageProvider ?? BundleImageProvider(bundle: bundle, searchPath: nil)
+    let provider = imageProvider ?? BundleImageProvider(bundle: bundle, searchPath: nil).cachedImageProvider
```

Comment:
If `BundleImageProvider` has its own internal cache, does it need to be wrapped in the cache wrapper? Does that mean that the image will be cached in two different places?

Reply:
It's an alternative implementation of https://github.com/airbnb/lottie-ios/pull/1590, and would like to make it easy to extent the cache capability for other image providers.